asv/repo.py

```diff
@@ -181,11 +181,11 @@ def get_repo(conf):
     if conf.dvcs is not None:
         for cls in util.iter_subclasses(Repo):
             if getattr(cls, 'dvcs') == conf.dvcs:
-                return cls(conf.repo, conf.project)
+                return cls(conf.repo, conf.mirror_dir)
```

Comment:
It probably doesn't matter, but it seems relatively cheap to maintain the old behavior here so we probably should.